tools/import_diagnostics.py

```diff
@@ -135,17 +239,33 @@ def zigpy_device_from_legacy_diagnostics(
                     real_cluster.unsupported_attributes.add(unsupported_attr)
 
         for cluster_id, cluster in ep["out_clusters"].items():
-            real_cluster = device.endpoints[int(epid)].out_clusters[int(cluster_id, 16)]
+            try:
+                real_cluster = device.endpoints[int(epid)].out_clusters[
+                    int(cluster_id, 16)
+                ]
+            except KeyError:
+                _LOGGER.warning(
+                    "Failed to find client cluster 0x%s on endpoint %s: %s",
+                    cluster_id,
+                    epid,
+                    cluster,
+                )
+                continue
+
             if patch_cluster:
                 patch_cluster_for_testing(real_cluster)
             for attr_id, attr in cluster["attributes"].items():
                 if (
-                    attr["value"] is None
+                    attr.get("value", None) is None
```

Comment:
nit: `None` should already be the default:
```suggestion
                    attr.get("value") is None
```